tools/constraints.txt

```diff
@@ -3,7 +3,7 @@
 # files because the version differs between them:
 # tensorflow, numpy
 mxnet~=1.2.0; sys_platform == 'win32'
-mxnet>=1.7.0.post2,<=1.9.1; sys_platform != 'win32'
+mxnet==1.9.1; sys_platform != 'win32'
```

Comment:
usually we use range of supported versions

Reply:
It's needed to increase the minimal required version to 1.9.1 and since it matches to the current maximal version only one choice left, to specify and exactly 1.9.1. The reason why upper bound is 1.9.1 because there are no new versions at the moment.

Reply:
we should still allow to install older version even they contains vulnerability bugs. 
It's a user choice what version to use. If he / she wants to use vulnerable version - up to them.
But by default OpenVINO will install latest version (from supported range of versions), which is ok from security perspective.

@rkazants @EfimovIlia what do you think?

Reply:
This topic has been already discussed several time about updating lower bound. And the conclusion is not to change it because security of user's platform is a responsibility of the user